<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>465</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>CCR Plus 测评器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/image.qrc">
    <normaloff>:/image/CCR.ico</normaloff>:/image/CCR.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	font-family: &quot;微软雅黑&quot;;
}
QMainWindow
{
	background-color:rgb(250,250,250);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_top">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;img src=&quot;:/image/logo2.png&quot;/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; color:#505050;&quot;&gt;最近打开的竞赛列表:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_recent">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget
{
	margin-left:-1px;
	margin-right:-1px;
}</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="gridSize">
         <size>
          <width>145</width>
          <height>150</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_bottom">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuContest">
    <property name="title">
     <string>竞赛(&amp;C)</string>
    </property>
    <widget class="QMenu" name="menu_recent">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>最近打开的竞赛(&amp;R)</string>
     </property>
    </widget>
    <addaction name="action_open"/>
    <addaction name="action_close"/>
    <addaction name="menu_recent"/>
    <addaction name="separator"/>
    <addaction name="action_configure"/>
    <addaction name="separator"/>
    <addaction name="action_set_list"/>
    <addaction name="action_export"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>测评(&amp;T)</string>
    </property>
    <addaction name="action_refresh"/>
    <addaction name="separator"/>
    <addaction name="action_judge_selected"/>
    <addaction name="action_judge_unjudged"/>
    <addaction name="action_judge_all"/>
    <addaction name="action_stop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_update"/>
    <addaction name="separator"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuContest"/>
   <addaction name="menuTest"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>新建或打开竞赛(&amp;O)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_close">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭当前竞赛(&amp;W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_refresh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>刷新列表(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>停止测评(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_judge_selected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>测评选中项目(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="action_judge_unjudged">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>测评未测项目(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="action_judge_all">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>测评所有项目(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F9</string>
   </property>
  </action>
  <action name="action_configure">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>配置试题(&amp;C)...</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_set_list">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>使用选手名单(&amp;L)...</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_export">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>导出成绩(&amp;E)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于 CCR Plus(&amp;A)...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>查看帮助(&amp;H)...</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_update">
   <property name="text">
    <string>检查更新(&amp;U)...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/image.qrc"/>
 </resources>
 <connections/>
</ui>
